### ISSUE TITLE
up/down arrow in integrated terminal prints escape sequence

### ISSUE THREAD

**Issue by 01binary** [microsoft/vscode#122802]
Issue Type: <b>Bug</b>

Press up arrow or down arrow keys while the focus is in the Integrated Terminal. Instead of cycling next/previous commands, it prints ^[[A or ^[[B to the terminal.

I am using VSCode on macOS 10.13.6

VS Code version: Code 1.55.2 (3c4e3df9e89829dce27b7b5c24508306b151f30d, 2021-04-13T09:36:32.643Z)
OS version: Darwin x64 17.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz (12 x 2900)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|32.00GB (8.85GB free)|
|Process Argv|--crash-reporter-id 816f329a-629d-4df9-ad82-46e1b2aaceb3|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (18)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-openapi|42C|4.5.0
vscode-apollo|apo|1.18.8
cmake-format|che|0.6.11
path-intellisense|chr|2.3.0
svg|joc|1.4.5
vscode-styled-components|jpo|1.5.2
openapi-lint|mer|1.2.0
csharp|ms-|1.23.11
vscode-ros|ms-|0.6.7
python|ms-|2021.4.765268190
jupyter|ms-|2021.5.745244803
remote-ssh|ms-|0.65.4
remote-ssh-edit|ms-|0.65.4
cpptools|ms-|1.3.1
vscode-typescript-tslint-plugin|ms-|1.3.3
vscode-yaml|red|0.18.0
mdx|sil|0.1.0
remote-x11-ssh|spa|1.5.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt678:30270856
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30276681
pythonvspyt639:30291489
pythontb:30283811
vspre833cf:30267465
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30276952
pythondataviewer:30285071
vscus158cf:30286554
vscgsv2:30294352

```

</details>

**Comment by 01binary** [comment]
When I went to provide additional information for this issue, I found a suggestion to enable "escape sequence logging". This is a debugging option for the terminal. Turns out it was actually enabled, that's why this was going on:

https://github.com/microsoft/vscode/wiki/Terminal-Issues#enabling-escape-sequence-logging

If you stumble on this issue when looking for a solution, hit F1 ("fn" key on newer Macs displays function keys in the touchbar) and type "Terminal: enable escape sequence logging". That will toggle it.
